app/src/main/java/tech/zafrani/companionforpubg/activities/DrawerActivity.java

```diff
@@ -16,70 +16,59 @@
 import android.view.MenuItem;
 import android.widget.FrameLayout;
 
+import butterknife.BindView;
 import tech.zafrani.companionforpubg.fragments.PUBGMapFragment;
 import tech.zafrani.companionforpubg.R;
 import tech.zafrani.companionforpubg.fragments.ItemFragment;
 import tech.zafrani.companionforpubg.utils.Constants;
 
-public abstract class DrawerActivity extends AppCompatActivity
+public abstract class DrawerActivity extends BaseActivity
         implements NavigationView.OnNavigationItemSelectedListener {
 
-    @Nullable
-    private DrawerLayout drawerLayout = null;
-    @Nullable
-    private NavigationView navigationView = null;
-    @Nullable
-    private FrameLayout contentLayout = null;
+    @BindView(R.id.activity_drawer_toolbar)
+    Toolbar toolbar;
+    @BindView(R.id.activity_drawer_drawerlayout)
+    DrawerLayout drawerLayout;
+    @BindView(R.id.activity_drawer_navigationview)
+    NavigationView navigationView;
 
     //region Activity
     @Override
     protected final void onCreate(@Nullable final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_drawer);
-
-        final Toolbar toolbar = (Toolbar) findViewById(R.id.activity_drawer_toolbar);
         setSupportActionBar(toolbar);
 
         final ActionBar actionBar = getSupportActionBar();
         if (actionBar != null) {
             actionBar.setHomeAsUpIndicator(R.drawable.ic_menu_white_24dp);
             actionBar.setDisplayHomeAsUpEnabled(true);
         }
-        this.contentLayout = (FrameLayout) findViewById(R.id.activity_drawer_content);
-        this.navigationView = (NavigationView) findViewById(R.id.activity_drawer_navigationview);
+
         this.navigationView.setNavigationItemSelectedListener(this);
```

Comment:
Something I forgot to do in my PR is remove this itemSelectedListener on destroy. Do you know if butterknife's unbind method will do that?